utilities/test_suite/HIP/runTests.py

```diff
@@ -33,6 +33,7 @@
 inFilePath2 = scriptPath + "/../TEST_IMAGES/three_images_mixed_src2"
 ricapInFilePath = scriptPath + "/../TEST_IMAGES/three_images_150x150_src1"
 qaInputFile = scriptPath + "/../TEST_IMAGES/three_images_mixed_src1"
+outFolderPath = scriptPath + "/../../../build"
```

Comment:
@kiritigowda The only assumption I make here is that the folder name is always "build" directly under "rpp". Is that okay?

Reply:
Fixed. Changed from build folder to CWD.